themes-default/slim/views/history.mako

```diff
@@ -220,9 +219,8 @@ const startVue = () => {
                 % for hItem in compactResults:
                     <tr>
                         <td align="center" class="triggerhighlight">
-                            <% airDate = sbdatetime.sbfdatetime(datetime.strptime(str(hItem.actions[0].date), History.date_format), show_seconds=True) %>
                             <% isoDate = datetime.strptime(str(hItem.actions[0].date), History.date_format).isoformat('T') %>
-                            <time datetime="${isoDate}" class="date">${airDate}</time>
+                            <time datetime="${isoDate}" title="${isoDate}" class="date">{{'${isoDate}' | formatDate}}</time>
```

Comment:
Tooltips should show date-times in a nice format IMO.

Reply:
The old timeago did exactly what I've got here so I didn't change it over.

Reply:
As we discussed, should be using a string representation of the date-time.